pkg/trace/writer/trace_test.go

```diff
@@ -74,6 +74,7 @@ func randomSampledSpans(spans, events int) *SampledSpans {
 
 // payloadContains checks that the given payload contains the given set of sampled spans.
 func payloadContains(t *testing.T, p *payload, sampledSpans []*SampledSpans) {
+	t.Helper()
```

Comment:
Interesting! Didn't know about this one.

Reply:
I also learned about it today: https://twitter.com/davidcrawshaw/status/1145344190806679552